Peggle/Peggle.toc

```diff
@@ -1,9 +1,10 @@
-## Interface: 80501
+## Interface: 80100
```

Comment:
note: 80100 is the current valid Interface version
![image](https://user-images.githubusercontent.com/964291/55238282-6652f300-5234-11e9-997e-51449aea4c2d.png)